binding_generator.py

```diff
@@ -151,7 +151,7 @@ def scons_generate_bindings(target, source, env):
         str(source[0]),
         env["generate_template_get_node"],
         "32" if "32" in env["arch"] else "64",
-        "double" if (env["float"] == "64") else "float",
+        "double" if env["precision"] == "double" else "float",
```

Comment:
This begs for maybe refactoring further and changing the values from `single|double` to `float|double` to match what is used in the GDExtension config:

```
gdextension/extension_api.json
12:                     "build_configuration": "float_32",
173:                    "build_configuration": "float_64",
334:                    "build_configuration": "double_32",
495:                    "build_configuration": "double_64",
658:                    "build_configuration": "float_32",
973:                    "build_configuration": "float_64",
1288:                   "build_configuration": "double_32",
1603:                   "build_configuration": "double_64",
```

Would need to be done upstream too and break compat slightly again, but might be worth it?